nixos/modules/services/x11/desktop-managers/lxqt.nix

```diff
@@ -15,25 +14,43 @@ in
 
   options = {
 
-    services.xserver.desktopManager.lxqt.enable = mkOption {
-      type = types.bool;
-      default = false;
-      description = "Enable the LXQt desktop manager";
+    services.xserver.desktopManager.lxqt.enable = mkEnableOption "the LXQt desktop manager";
+
+    services.xserver.desktopManager.lxqt.enableWaylandSession = mkEnableOption "the experimental LXQt Wayland session";
+
+    services.xserver.desktopManager.lxqt.iconThemePackage =
+      lib.mkPackageOption pkgs [ "kdePackages" "breeze-icons" ] { } // {
+        description = "The package that provides a default icon theme.";
+      };
+
+    services.xserver.desktopManager.lxqt.extraPackages = lib.mkOption {
+      type = with lib.types; listOf package;
+      default = with pkgs; [ openbox lxqt.obconf-qt ];
```

Comment:
I am not sure if this is a good design, people will be suprised that openbox is gone as soon as they set extraPackages themselves.